index.html

```diff
@@ -1,51 +1,77 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
 </head>
+
 <body>
-   
 
-<div id="container">
 
 
 
-<img id="mylogo" src="/Portfolio-setup-and-mobile-version-skeleton/Image/Logo david.jpg" alt="">
-<div class="Header">
+  <section class="top-nav">
+    <div>
+      <img id="mylogo" src="/Image/logo-david-nuevo-2021.png" alt="">
+    </div>
+    <label for="menu-toggle">MY FIRST WEB PAGE</label>
+    <input id="menu-toggle" type="checkbox" />
+    <label class='menu-button-container' for="menu-toggle">
+      <span class='menu-button'></span>
+    </label>
+    <ul class="menu">
+      <li>Portfolio</li>
+      <li>About</li>
+      <li>Contact</li>
+
+
+    </ul>
+  
+  </section>
+  <h1> </h1>
+  <img id="img11" src="/Image/Rectangle6.svg" alt="">
+  <img id="img1" src="/Image/Ellipse11.svg" alt="">
+  <img id="img2" src="/Image/Ellipse5.svg" alt="">
+  <img id="img3" src="/Image/Ellipse8.svg" alt="">
+  <img id="img4" src="/Image/Ellipse3.svg" alt="">
+  <img id="img5" src="/Image/Rectangle7.svg" alt="">
+  <img id="img6" src="/Image/Rectangle4.svg" alt="">
+  <img id="img7" src="/Image/Ellipse8.svg" alt="">
+  <img id="img8" src="/Image/Ellipse8.svg" alt="">
+  <img id="img9" src="/Image/Rectangle7.svg" alt="">
+  <img id="img0" src="/Image/Ellipse3.svg" alt="">
 
-</div>
-    <nav>
-<ul>
-   <li>Portfolio</li>
-   <li>About</li>
-   <li>Contact</li>
-</ul>
-</nav>
-</div>
 
-<pre id="name">I'm David Verde   
-    Glad to see you! 
+  <pre id="name">I'm David Verde! 
+Glad to see 
+you! 
 </pre>
-<pre id="content">
-I'm a software developer! I can help you build a product , feature or website Look through
-some of my work and experience! If you like what you see and have a project you need 
-coded, don't hestiate to contact me. 
+  <pre id="content">
+I'm a software developer! I can help you
+build a product , feature or website Look 
+through some of my work and
+experience! If you like what you see and
+have a project you need coded, don't
+hestiate to contact me. 
 </pre>
 
-<h3 id="connect">
+  <h3 id="connect">
     LET'S CONNECT
 
-</h3>
-<div class="icons">
-<img class="social" src="/Portfolio-setup-and-mobile-version-skeleton/Image/twitter.svg" alt="">
-<img id="Vector" src="/Portfolio-setup-and-mobile-version-skeleton/Image/Vector.svg" alt="">
-<img class="social" src="/Portfolio-setup-and-mobile-version-skeleton/Image/medium.svg" alt="">
-<img class="social" src="/Portfolio-setup-and-mobile-version-skeleton/Image/github.svg" alt="">
-<img class="social" src="/Portfolio-setup-and-mobile-version-skeleton/Image/angellist.svg" alt="">
-</div>
+  </h3>
+  <div class="icons">
+    <img src="/Image/twitter.svg" alt="">
+    <img id="Vector" src="/Image/Vector.svg" alt="">
+    <img src="/Image/medium.svg" alt="">
+    <img src="/Image/github.svg" alt="">
+    <img src="/Image/angellist.svg" alt="">
+
+
+  </div>
```

Comment:
Please use `<ul>` `<li>` tags to display list of icons and use `<a>` for links.
#### Example
```
<ul>
  <li>
    <a href="github.com">
      <img src="github.jpg" alt=""/>
    </a>
  </li>
....
....
.....
</ul>
```